src/Uno.UI.Toolkit/ElevatedView.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media;
+#if __IOS__
+using CoreGraphics;
+using _View = UIKit.UIView;
+#elif __MACOS__
+using CoreGraphics;
+using _View = AppKit.NSView;
+#endif
+
+
+namespace Uno.UI.Toolkit
+{
+	[ContentProperty(Name = "Content")]
+	[TemplatePart(Name = "PART_Border", Type = typeof(Border))]
+	public sealed partial class ElevatedView : Control
+#if !NETFX_CORE
+		, ICustomClippingElement
+#endif
+	{
+		/*
+		 *  +-ElevatedView------------+
+		 *  |                         |
+		 *  |  +-Border------------+  |
+		 *  |  |                   |  |
+		 *  |  |  +-Content-----+  |  |
+		 *  |  |  | (...)       |  |  |
+		 *  |  |  +-------------+  |  |
+		 *  |  |                   |  |
+		 *  |  +-------------------+  |
+		 *  |                         |
+		 *  +-------------------------+
+		 *
+		 * Elevated is responsible for the shadow
+		 * Border responsible for rounded corners (if any)
+		 *
+		 */
+
+		private Border _border = new Border();
+
+		public ElevatedView()
+		{
+			DefaultStyleKey = typeof(ElevatedView);
+		}
+
+		protected override void OnApplyTemplate()
+		{
+			_border = GetTemplateChild("PART_Border") as Border;
+			Console.WriteLine($"ApplyTemplate - border={_border}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5093276)